c8run/package.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"c8run/internal/archive"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+func Clean(camundaVersion string, elasticsearchVersion string) {
+	os.RemoveAll("elasticsearch-" + elasticsearchVersion)
+	os.RemoveAll("camunda-zeebe-" + camundaVersion)
+
+	_, err := os.Stat(filepath.Join("log", "camunda.log"))
+	if errors.Is(err, os.ErrNotExist) {
+		os.Remove(filepath.Join("log", "camunda.log"))
+	}
+	_, err = os.Stat(filepath.Join("log", "connectors.log"))
+	if errors.Is(err, os.ErrNotExist) {
+		os.Remove(filepath.Join("log", "connectors.log"))
+	}
+	_, err = os.Stat(filepath.Join("log", "elasticsearch.log"))
+	if errors.Is(err, os.ErrNotExist) {
+		os.Remove(filepath.Join("log", "elasticsearch.log"))
+	}
+}
+
+func downloadAndExtract(filePath, url, extractDir string, extractFunc func(string, string) error) error {
+	_, err := os.Stat(filePath)
+	if errors.Is(err, os.ErrNotExist) {
+		err = archive.DownloadFile(filePath, url)
```

Comment:
"download if not exist" can be part of download archive

Reply:
I've pushed that logic into DownloadFile.